integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ConversationMarker.java

```diff
@@ -0,0 +1,17 @@
+package no.difi.meldingsutveksling.receipt;
+
+import net.logstash.logback.marker.LogstashMarker;
+import net.logstash.logback.marker.Markers;
+
+/**
+ * Creates markers for Conversations.
+ */
+public class ConversationMarker {
+    private static final String MESSAGE_ID = "message_id";
+
+    public static LogstashMarker markerFrom(Conversation conversation) {
+        final LogstashMarker messageIdMarker = Markers.append(MESSAGE_ID, conversation.getConversationId());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "messageIdMarker". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1488)